pom.xml

```diff
@@ -103,15 +107,25 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>org.jeasy</groupId>
+            <artifactId>easy-rules-core</artifactId>
+            <version>4.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jeasy</groupId>
+            <artifactId>easy-states</artifactId>
+            <version>2.0.0</version>
+        </dependency>
         <dependency>
             <groupId>net.imagej</groupId>
             <artifactId>ij</artifactId>
             <version>1.53h</version>
         </dependency>
         <dependency>
-            <groupId>net.jthink</groupId>
+            <groupId>com.github.goxr3plus</groupId>
```

Comment:
From eigenholser in #24 : `TODO:` Would like to find a more canonical source for this.